pkg/runtime-tools/generate/generate.go

```diff
@@ -323,6 +334,20 @@ func (g *Generator) AdjustDevices(devices []*nri.LinuxDevice) {
 	}
 }
 
+// InjectCDIDevices injects CDI devices to the OCI Spec.
```

Comment:
It's not clear how this injects the device into the OCI spec specifically.

Reply:
Updated comment to be more specific about how actual CDI device injection happens.